consensus/types/src/fork_name.rs

```diff
@@ -201,6 +201,46 @@ impl ForkName {
     pub fn gloas_enabled(self) -> bool {
         self >= ForkName::Gloas
     }
+
+    pub fn fork_ascii(self) {
+        if self == ForkName::Fulu {
+            println!(
+                r#"
+                                  ╔═══════════════════════════════════════╗
+                                  ║                                       ║
+                                  ║   TO FULU, MOAR BLOBS TO ETHEREUM     ║
+                                  ║                                       ║
+                                  ║        III DECEMBER MMXXV             ║
```

Comment:
```suggestion
                                  ║          III DECEMBER MMXXV           ║
```
center, 10 chars left 11 right

Reply:
centered